src/web/modules/ledger_payouts/list.njk

```diff
@@ -0,0 +1,112 @@
+{% extends "layout/layout.njk" %}
+
+{% macro payout_status(status) %}
+  <strong class="govuk-tag
+        {% if (status=='paidout') %} status-success {% endif %}
+        {% if (status=='failed') %} status-failed {% endif %}
+        {% if (status=='intransit') %} status-in-progress {% endif %}
+      status-tag">
+    <span>{{ status | capitalize }}</span>
+  </strong>
+{% endmacro %}
+
+{% block main %}
+
+  <span class="govuk-caption-m">
+    {% if service %}
+      <span>{{ service.name }}</span>
+    {% else %}
+      <span>GOV.UK Pay platform</span>
+    {% endif %}
+  </span>
+
+  <h1 class="govuk-heading-m">Payouts</h1>
+
+  {% if accountId %}
+    <div class="govuk-body">
+      <a href="/ledger_payouts">
+        <span>View all payouts</span>
+      </a>
+    </div>
+  {% endif %}
+
+  <div class="transactions-filter__container govuk-body">
+    {% for status in["intransit", "paidout", "failed", "all"] %}
+      <a
+        class="transactions-filter__item {% if selectedStatus === status %}selected{% endif %} no-decoration"
+        href="/ledger_payouts?status={{ status }}{% if service %}&account={{ accountId }}{% endif %}">
+        <span>{{ status | capitalize }}</span>
+      </a>
+    {% endfor %}
+  </div>
+
+  <div>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
```

Comment:
Some UX tweaks for things that might clean the table up and let everything be shown on one line:

* limit the payout ID string to something like 8 characters using `| truncate(8)`, I think most of the time the full ID won't be needed. Using `<span title="the-full-payout-id">` would allow users to see the full ID if hovered
* the transactions view has the status badge next in-line as it's visually clear what it indicates, doing this would allowing removing the 'State' column (@iqbalgds mentioned there could be accessibility considerations about making sure this is labeled properly)
* I think the main bit of information for paid out is the date itself, it could be that we could format it to remove the time to reduce information overload
* Removing the gateway account link from the beginning, as this is just a number it's not immediately clear what it refers to
* Doing this might allow an "Actions" table header at the end which could have 'Account' and 'Transactions' links